content/pages/disability-benefits/apply/claim-types/fully-developed-claim-alt.md

```diff
@@ -0,0 +1,116 @@
+<li>
+<button class="usa-button-unstyled usa-accordion-button" aria-controls="need-to-do">What do I need to do?</button>
+<div id="need-to-do" class="usa-accordion-content">
+
+#### FDC Program
+
+When turning in your claim under the FDC program, please send us all the supporting documents you have and provide information about supporting documents that you don’t have.
+
+1. [Fill out an Application for Disability Compensation (VA Form 21P-526EZ)](/disability-benefits/apply/).<br>
+2. When you file your claim, you'll need to turn in:
+    - A copy of your DD214 or other separation documents,  **and**
+    - Service treatment records,  **and**
+    - All private and VA medical records related to your illness or injury, **and**
+    - Information about any health records that are available at a federal facility, like a VA medical center or clinic
+3. Go to any VA medical appointments required to decide your claim.
+
+
+<br>
+
+#### Standard Claim Process
+
+With a standard claim, you’ll give us permission to gather evidence or records for you. You’ll need to give us enough information about the evidence so we can request it from the right person or agency.
+
+If a person or agency refuses to turn over the evidence, asks for money for the evidence, or can’t get the evidence, we’ll ask you to try to get the evidence or records. It’s your responsibility to make sure we receive all requested records that aren’t at a federal agency or office.
+
+</div>
+</li>
+<li>
+<button class="usa-button-unstyled usa-accordion-button" aria-controls="va-will-help">How will VA help me get evidence for my claim?</button>
+<div id="va-will-help" class="usa-accordion-content">
+
+#### FDC Program
+
+Under the FDC program, we’ll handle the evidence-gathering steps listed below. 
+
+**We’ll:**
+
+* Request relevant records that you identify and authorize us to get from a federal facility, like a VA medical center
+* Schedule a health exam or get a medical opinion for you, if we decide we need it for your claim
+
+<br>
+
+#### Standard Claim Process
+
+With the standard claim process, we’ll handle the evidence-gathering steps listed below.
+
+**We'll:**
+
+* Request relevant records that you identify and authorize us to get from a federal facility, like a VA medical center
+* Schedule a health exam or get a medical opinion for you, if we decide we need it for your claim
+* Make every reasonable effort to request relevant records that you authorize us to get that aren’t at a federal facility. These may include records from state or local governments, documents from a private doctor or hospital, or records from current or former employers.
+
+</div>
+</li>
+<li>
+<button class="usa-button-unstyled usa-accordion-button" aria-controls="when-to-send">When should I turn in my evidence?</button>
+<div id="when-to-send" class="usa-accordion-content">
+
+#### FDC Program
+
+You’ll need to turn in the information and evidence at the same time as you file your claim.
```

Comment:
Should this be "You’ll need to provide the information and evidence at the same time that you file your claim." or perhaps "while/when filing your claim"?